packages/vue-components/src/Searchbar.vue

```diff
@@ -121,7 +121,7 @@ export default {
           headingKeywords,
         } = entry;
         const keywords = entry.keywords || '';
-        const displayTitle = title || src;
+        const displayTitle = title || src.substring(0, src.indexOf('.'));
```

Comment:
What happens if I have a file that includes '.' in the filename?
e.g.
```
const src = "test.hello.md"
console.log(src.substring(0, src.indexOf('.'))); // output "test"
```

Reply:
brain glitch :') I forgot that files can have '.' in the name.